browser/ui/brave_browser_command_controller.cc

```diff
@@ -260,6 +266,16 @@ void BraveBrowserCommandController::UpdateCommandForBraveVPN() {
 #endif
 }
 
+void BraveBrowserCommandController::UpdateCommandForPlaylist() {
+#if BUILDFLAG(ENABLE_PLAYLIST_WEBUI)
+  if (base::FeatureList::IsEnabled(playlist::features::kPlaylist)) {
+    UpdateCommandEnabled(
+        IDC_SHOW_PLAYLIST_BUBBLE,
+        browser_->is_type_normal() && !browser_->profile()->IsOffTheRecord());
```

Comment:
do not duplicate checks like this, you should be checking to see if the profile is null from the factory